src/App.svelte

```diff
@@ -229,10 +229,10 @@
   <WalletConnectModal />
   <main>
     <Header />
-    <Route path="/activerewards" primary={false}>
+    <Route path="/rewards" primary={false}>
       <ClaimPortal />
     </Route>
-    <Route path="/datafarming" primary={false}>
+    <Route path="/volumedf" primary={false}>
```

Comment:
1. I think we should do these as part of separate update.
Changing routes can break all sorts of things, so we need to do a seprate pass and verify everything is alright.

2. I don't recommend us changing from datafarming => volumedf... Volume df is an internal name. As documented in the ticket.

Reply:
It's a very small change so I wouldn't open another PR for that. This PR should be merged only when we are ready to launch so this implies that we should check nothing it's broken.

We should change from datafarming since now all of the substreams are part of data farming. We could use another name but I'd like to be consistent, so the name we are displaying should be the name used internally so we It won't cause misunderstandings or mistakes.